docs/dietpi_docs_todo.md

```diff
@@ -105,19 +105,31 @@ The software list between the softwares listed in
 
 need to be checked for consistency.
 
+#### Result
+
 Softwares which are actually not within `dietpi-software`:
 
-- t.b.d.
+- [jRiver](https://dietpi.com/docs/software/media/#jriver-media-center)
+- [OpenTyrian](https://dietpi.com/docs/software/gaming/#opentyrian-gaming)
+- [PaperMC](https://dietpi.com/docs/software/gaming/#papermc)
+- [Raspcontrol](https://dietpi.com/docs/software/system_stats/#raspcontrol-web-interface-system-stats)
+- [Grasshopper](https://dietpi.com/docs/software/home_automation/#grasshopper-web-app-to-control-bticino-myhome)
```

Comment:
- OpenTyrian is still available:
    ```
    2020-11-19 17:23:36 root@micha:/var/log# dietpi-software list | grep tyrian
    id 51 | =0 | opentyrian: a classic retro game, addictive | +alsa +xserver | https://dietpi.com/phpbb/viewtopic.php?p=45#p45
    ```
- `jRiver` has been removed with v6.12 indeed. We still have an autostart option for it, but probably we should remove that as well as we cannot test with an own implementation.
- `PaperMC` comes new with v6.34.
- `Raspcontrol` and `Grasshopper` have been removed with v6.2 indeed.

Reply:
Ok, i.e. PaperMC is kept as an open point.